docs/content/configuration/index.md

```diff
@@ -783,8 +783,8 @@ A sample Coordinator dynamic config JSON object is shown below:
   "replicationThrottleLimit": 10,
   "emitBalancingStats": false,
   "killDataSourceWhitelist": ["wikipedia", "testDatasource"],
-  "historicalNodesInMaintenance": ["localhost:8182", "localhost:8282"],
-  "nodesInMaintenancePriority": 7
+  "decommissionNodes": ["localhost:8182", "localhost:8282"],
+  "decommissionPriority": 7
```

Comment:
Parhaps better to call it "decommisioningVelocity" as @drcrallen suggested here: https://github.com/apache/incubator-druid/pull/6349#discussion_r224145700 because "priority" is a super overloaded term.

Reply:
Makes sense to me, though I think the properties should be consistent named so maybe I'll either rename the other `decommissioningNodes` or call this one `decommissionVelocity`  to match. Anyone have a preference?

Reply:
Renamed to `decommissioningNodes` and `decommissioningVelocity`